docs/adr/0009_versioning_strategy.md

```diff
@@ -0,0 +1,299 @@
+
+## Considered Options
+
+I explicitly considered the following options before making the decision above. Recording them here improves traceability.
+
+### Versioning options
+
+* Per-component `VERSION` file: each component contains a `VERSION` file in its directory that is the working state for that component.
+* Central `VERSIONS.yaml`: a single root `VERSIONS.yaml` that is the canonical snapshot of component versions (human-edited or bot-maintained).
+* Hybrid: per-component `VERSION` files for working state, plus a generated `/VERSIONS.yaml` for published snapshots.
+
+### Branching options
+
+* Option A — Persistent component-scoped release branches: `releases/<component>/v<major>.<minor>` maintained as long-lived maintenance lines.
+* Option B — Tag-only / short-lived release PRs: create ephemeral release-candidate branches or push tags directly; do not maintain long-lived release branches.
+
+### Compatibility matrix (high-level)
+
+| Versioning \\\ Branching | Option A (persistent branches) | Option B (tag-only / short-lived) |
+| ----------------------: | :----------------------------: | :-------------------------------: |
+| Per-component `VERSION`  | Good — native fit: branches host `VERSION`, easy hotfixes | OK — works, but hotfixes require creating temporary branches; more manual steps |
+| Central `VERSIONS.yaml`  | OK — CI must reconcile branch-local changes into central file; potential conflicts | Fair — central file updated from tags only; high merge-conflict risk if many concurrent releases |
+| Hybrid (chosen)         | Best — combines autonomy and a single snapshot; CI can generate `/VERSIONS.yaml` on publish | Good — possible but increases CI complexity to update central snapshot safely |
+
+### Rejection rationale (short)
+
+* Central-only `VERSIONS.yaml` + tag-only branching: rejected because hotfix/backport workflows become cumbersome and the central file becomes a high-conflict surface.
+* Per-component `VERSION` without central snapshot: rejected because consumers (website, docs) need a single reproducible snapshot; lacking that reduces discoverability and reproducibility of root releases.
+
+## Operational Rules and Rationale
+
+### Authoritative sources
+
+* Working state (pre-publication): `ocm-<component>/VERSION` located on the appropriate branch (either `main` for ongoing development, or a `releases/...` branch for maintenance lines).
+* Published state: annotated Git tag `<component>/v<major>.<minor>.<patch>` — tags are authoritative for published artifacts and their provenance.
+* Global snapshot: `/VERSIONS.yaml` — generated by CI after successful publishes and updated via OCM bot PRs; **this file is not intended for manual edits**.
+
+Rationale: Having both per-component `VERSION` files (working state) and a generated `/VERSIONS.yaml` provides autonomy and a single snapshot for consumers.
```

Comment:
how does this provide autonomy?

Reply:
+1 for this, but I think you could state this explicitly (for me it was not clear right away)